dsd/vienna_nupack.py

```diff
@@ -93,6 +101,42 @@ def pfunc(seqs: Union[str, Tuple[str, ...]],
 
     return -dg if negate else dg
 
+@lru_cache(maxsize=10_000)
```

Comment:
Add to the docstring here (and in the original pfunc, where I was lazy and should have done it already) indicating what are the installation requirements to call the function. In this case, that the Python package needs to be installed along with NUPACK 4. In the original, that NUPACK version 2 or 3 must be installed and on the PATH.